src/pages/Quote.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] Let's name disable linter rules. these are here to help us write a clean block of code.